npm/vue/src/index.ts

```diff
@@ -1,17 +1,43 @@
 /* eslint-disable no-redeclare */
 /// <reference types="cypress" />
-import { ComponentPublicInstance, VNodeProps, AllowedComponentProps,
-  ComponentCustomProps, ExtractPropTypes, ExtractDefaultPropTypes,
-  Component, DefineComponent, FunctionalComponent, ComputedOptions,
-  MethodOptions, ComponentOptionsMixin, EmitsOptions, ComponentOptionsWithObjectProps, ComponentPropsOptions, ComponentOptionsWithArrayProps, ComponentOptionsWithoutProps } from 'vue'
-import { MountingOptions, VueWrapper, mount as VTUmount } from '@vue/test-utils'
+import type {
+  ComponentPublicInstance,
+  VNodeProps,
+  AllowedComponentProps,
+  ComponentCustomProps,
+  ExtractPropTypes,
+  ExtractDefaultPropTypes,
+  DefineComponent,
+  FunctionalComponent,
+  ComputedOptions,
+  MethodOptions,
+  ComponentOptionsMixin,
+  EmitsOptions,
+  ComponentOptionsWithObjectProps,
+  ComponentPropsOptions,
+  ComponentOptionsWithArrayProps,
+  ComponentOptionsWithoutProps,
+} from 'vue'
+import type { MountingOptions, VueWrapper } from '@vue/test-utils'
 import {
   injectStylesBeforeElement,
   StyleOptions,
   getContainerEl,
   setupHooks,
 } from '@cypress/mount-utils'
 
+import * as _VueTestUtils from '@vue/test-utils'
+
+const {
+  mount: VTUmount,
+  // TODO: Should we expose shallowMount? It doesn't make much sense in the context of Cypress
```

Comment:
Is this really a TODO anymore? Seems like maybe just a comment is needed to explain why we're exporting even though it doesn't make a ton of sense for Cy

Reply:
Yeah I totally agree with exposing it for easy migration and keeping tests as close as possible before and after. Folks can modify their tests as a separate step when they have the need to see their components render fully.

Reply:
Look carefully - we are **not** exposing it:

```js
const {
  mount: VTUmount,
  // TODO: Should we expose shallowMount? It doesn't make much sense in the context of Cypress
  // but it might be useful for people who like to migrate some Test Utils tests to Cypress.
  shallowMount,
  ...VueTestUtils
} = _VueTestUtils

export { VueTestUtils }
```

We destructure the properties we **do not** want to expose, and the rest are in the remaining `...VueTestUtils` object, which we export.

We should not expose `shallowMount` imo, since Cypress is supposed to encourage production-like testing, and `shallowMount` leads to all kind of weird, impossible-in-production scenarios.

We could expose it for migration purposes, I'd prefer to wait for someone to ask for this, though. I generally think we shouldn't expose it, since

- React and Angular don't have something similar
- If you do a `shallowMount`, most of Cypress (`cy.get` etc) won't work, since everything is stubbed out.

Reply:
@mike-plummer I will remove TODO and just say the reason we don't expose it.

Reply:
Done!